### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by santiagorodriguezbermudez**
### On these tests we worked on:
- [x] Run the project using Webpack
- [x] Tested the player factory object to output the player objects accordingly
- [x] Tested the game board to update the moves Array accordingly
- [x] Test the game board to reset its own moves array accordingly
- [x] Tested the game logic to update the tokens array accordingly

### CODE REVIEW

File: src/test/game.test.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import GameLogic from '../js/game';
```
Comment (christianotieno) [inline src/test/game.test.js:1]: ## Suggested requirements:
In our game logic, kindy let's add tests to check for: 👇 
- [x] Draw.
- [x] Game reset.
- [x] Player input.
- [x] Winning conditions.
- [x] Invalid move by the player.

**Review by christianotieno — CHANGES_REQUESTED** [review]
# Changes Required ♻️ 
Hello team 👋 . Good job with the project.
- [x]  Update the README documentation with instructions on how well to clone, install dependencies, run the code, and tests locally, thank you. 
- [x] The test suites we have are currently not sufficiently testing the entire game logic, we needed to add more test, currently, look at the suggestions below 👇 
#### Christian Otieno (TSE)

**Comment by LoboArkano** [comment]
We can't test private functions, but we test the effects of the functions.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
***Thank you for putting into consideration the earlier recommended changes.  🙏 All the rest of the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job :+1:
- You can now merge these changes.
- I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**